src/data/gui.rs

```diff
@@ -21,17 +21,17 @@ use crate::validator::Validator;
 #[derive(Debug, Default)]
 pub struct Gui {
     files: FnvHashMap<PathBuf, Vec<GuiWidget>>,
-    templates: FnvHashMap<String, GuiTemplate>,
+    templates: FnvHashMap<&'static str, GuiTemplate>,
     // Type keys are stored in lowercase because type lookup is case-insensitive
-    types: FnvHashMap<String, GuiType>,
-    layers: FnvHashMap<String, GuiLayer>,
+    types: FnvHashMap<Lowercase<'static>, GuiType>,
+    layers: FnvHashMap<&'static str, GuiLayer>,
     // TextIcon is in a Vec because a single icon can have multiple definitions with different
     // iconsize parameters.
-    texticons: FnvHashMap<String, Vec<TextIcon>>,
-    textformats: FnvHashMap<String, TextFormat>,
+    texticons: FnvHashMap<&'static str, Vec<TextIcon>>,
+    textformats: FnvHashMap<&'static str, TextFormat>,
     // This is indexed by a (colorblindmode, textformatname) pair
-    textformats_colorblind: FnvHashMap<(String, String), TextFormat>,
-    widget_names: FnvHashMap<String, Token>,
+    textformats_colorblind: FnvHashMap<(&'static str, &'static str), TextFormat>,
+    widget_names: FnvHashMap<&'static str, Token>,
```

Comment:
Probably can use Hashset now that Token implements Borrow<str>

Reply:
done :)  I had to rebase to get the Borrow implementation in